padrino-helpers/lib/padrino-helpers/render_helpers.rb

```diff
@@ -34,7 +34,7 @@ def partial(template, options={}, &block)
         options = { :locals => {}, :layout => false }.update(options)
         explicit_engine = options.delete(:engine)
 
-        path,_,name = template.to_s.rpartition(File::SEPARATOR)
+        path, name = File.split(template.to_s)
         template_path = File.join(path,"_#{name}").to_sym
```

Comment:
It it an actual problem you encounter or a theoretical one?

I'm asking because according to the code before the commit it would call `render(explicit_engine, :'/hello', options)`, not `render(explicit_engine, :hello, options)`.

I think :'./hello' more precisely represents :hello template than :'/hello' which can be viewed as an absolute path.